examples/auth0/lib/auth0.js

```diff
@@ -0,0 +1,14 @@
+import { useAuth0 } from '@auth0/nextjs-auth0'
```

Comment:
The `use` here could be taken as implying this is a React hook which it's not. I would suggest changing the name to something else. Potentially `initAuth0` or something.

Reply:
upstream PR [here](https://github.com/auth0/nextjs-auth0/pull/10)